prow/config/templates/ing-Ingress.yaml

```diff
@@ -24,13 +24,6 @@ spec:
   - host: {{ .Values.prow.domain }}
     http:
       paths:
-      - path: /*
-        pathType: ImplementationSpecific
-        backend:
-          service:
-            name: deck
-            port:
-              number: 80
       - path: /hook
```

Comment:
Mind putting a comment in here please that states the order of these paths is important? I'd hate to have someone revisit this in a year and break things not knowing that the order was significant...